Octokit/Clients/SearchClient.cs

```diff
@@ -83,5 +83,18 @@ public Task<SearchLabelsResult> SearchLabels(SearchLabelsRequest search)
             Ensure.ArgumentNotNull(search, nameof(search));
             return ApiConnection.Get<SearchLabelsResult>(ApiUrls.SearchLabels(), search.Parameters);
         }
+
+        /// <summary>
+        /// search commits
+        /// https://developer.github.com/v3/search/#search-commits
+        /// </summary>
+        /// <param name="search"></param>
+        /// <returns>List of commits</returns>
+        [ManualRoute("GET", "/search/commits")]
+        public Task<SearchCommitsResult> SearchCommits(SearchCommitsRequest search)
+        {
+            Ensure.ArgumentNotNull(search, nameof(search));
+            return ApiConnection.Get<SearchCommitsResult>(ApiUrls.SearchCommits(), search.Parameters, @"application/vnd.github.cloak-preview");
```

Comment:
The `application/vnd.github.cloak-preview` header should be in `Octokit/Helpers/AcceptHeaders.cs`.

Are you able to do these two things to be consistent with the rest of the codebase:

 - add a const for this header in `Octokit/Helpers/AcceptHeaders.cs`
 - add this as a valid preview header in the convention test so the build doesn't fail

https://github.com/octokit/octokit.net/blob/e41b2d19d74d84e5b5222ebce58d64fc308b1c2e/Octokit.Tests.Conventions/PreviewsTests.cs#L11-L21

Reply:
I created the constant.  I also created a new array for non standard preview headers in the test since this header does not have "+json" at the end.